modules/sdk-coin-apt/src/apt.ts

```diff
@@ -103,4 +180,8 @@ export class Apt extends BaseCoin {
   signTransaction(params: SignTransactionOptions): Promise<SignedTransaction> {
```

Comment:
this method doesn't have unit tests. Refer modules/sdk-coin-hbar/test/unit/hbar.ts.